scipy/stats/tests/test_distributions.py

```diff
@@ -261,6 +262,13 @@ def _assert_less_or_close_loglike(dist, data, func=None, **kwds):
 
     mle_analytical = dist.fit(data, **kwds)
     numerical_opt = super(type(dist), dist).fit(data, **kwds)
+
+    # Sanity check that the analytical MLE is actually executed.
+    # Due to floating point arithmetic, the generic MLE is unlikely
+    # to produce the exact same result as the analytical MLE.
+    if not maybe_identical:
```

Comment:
`if maybe_identical is True` easy to read and more udnerstandable

Reply:
Do you mean `if maybe_identical is False`?

Reply:
Yes exactly sorry :-) But this is the point. `if maybe_identical is False` is much easier to understand

Reply:
The general recommendation from [Python style guide](https://peps.python.org/pep-0008/) is “Don’t compare boolean values to True or False using == [or is]”. Though in this case I agree “not” followed by “maybe” sounds like double negation and not entirely straightforward. Since this is just in the test and not in the main code, I’d prefer to leave it as is to follow the style guide.